deployment/environments/dev/values.yaml

```diff
@@ -1,23 +1,19 @@
 networks:
   - name: polygon-mumbai
-    rpc: https://rpc-mumbai.maticvigil.com/
-    host: polygon-mumbai.unode.gorengine.com
+    host: polygon-mumbai-staging.unode.gorengine.com
     debug: true
-    startingBlock: "0"
+    startingBlock: 42400000
```

Comment:
I've looked for the ownership block of each ownership chain right after the first evo block of klaos.
These are the ones:
polygon mainnet -> **50442008**
polygon mumbai -> **42891324**
ethereum mainnet -> **18661200**

Please, replace the values accordingly in all the `.yaml` files

Reply:
Thanks for these numbers. I will put these numbers as starting blocks. I will put actually 10 blocks before. It won't change much but I like to have ownership starts before evo